tools/setup_perf_cluster.sh

```diff
@@ -0,0 +1,264 @@
+#! /bin/bash
+#
+# Sets up a cluster for perf testing - GCP/GKE
+# PROJECT=istio-perf tools/setup_perf_cluster.sh
+#
+# This can be used as a script or sourced and functions called interactively
+#
+
+ZONE=${ZONE:-us-east4-b}
+CLUSTER_NAME=${CLUSTER_NAME:-istio-perf}
+MACHINE_TYPE=${MACHINE_TYPE:-n1-highcpu-2}
+NUM_NODES=${NUM_NODES:-6} # SvcA<->SvcB + Ingress + Pilot + Mixer + 1 extra (kube-system)
+VM_NAME=${VM_NAME:-fortio-vm}
+ISTIOCTL=${ISTIOCTL:-istioctl} # to override istioctl from outside of the path
+FORTIO_NAMESPACE=${FORTIO_NAMESPACE:-fortio} # Namespace for non istio app
+ISTIO_NAMESPACE=${ISTIO_NAMESPACE:-istio} # Namespace for istio injected app
```

Comment:
```
### Running: sh -c istioctl kube-inject --debug=false -n istio -f ./tools/perf_k8svcs.yaml > ./tools/perf_k8svcs_istio.yaml
### Running: kubectl apply -n istio -f ./tools/perf_k8svcs_istio.yaml
Error from server (NotFound): error when creating "./tools/perf_k8svcs_istio.yaml": namespaces "istio" not found
Error from server (NotFound): error when creating "./tools/perf_k8svcs_istio.yaml": namespaces "istio" not found
Error from server (NotFound): error when creating "./tools/perf_k8svcs_istio.yaml": namespaces "istio" not found
Error from server (NotFound): error when creating "./tools/perf_k8svcs_istio.yaml": namespaces "istio" not found
Error from server (NotFound): error when creating "./tools/perf_k8svcs_istio.yaml": namespaces "istio" not found
```

```
➜  istio git:(perf_scripts) ✗ k get ns
NAME           STATUS    AGE
default        Active    5m
fortio         Active    4m
istio-system   Active    4m
kube-public    Active    5m
kube-system    Active    5m
```

Should it be istio-system or something?

Reply:
good catch, thank you - somehow the create namespace for that one is missing

Reply:
it's commented out on line 114 so it doesn't fail on rerun